wcc-contentful-app/Gemfile

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# Specify your gem's dependencies in wcc-contentful.gemspec
+gemspec
+
+gem 'wcc-contentful', path: File.expand_path('../wcc-contentful', __dir__)
```

Comment:
doing this over dependency in gemspec?

Reply:
This is the trick for getting local development to work automagically.  The gemspec should never assume the path of the dependent `wcc-contentful`, I want Bundler to handle that when we install into Watermark Resources.